nilearn/datasets/utils.py

```diff
@@ -16,6 +16,11 @@
 import zipfile
 from .._utils.compat import _basestring, cPickle, _urllib, md5_hash
 
+# Environment variables splitter is different on Windows
+if sys.platform == 'win32':
+    ENV_SPLITTER = ';'
+else:
+    ENV_SPLITTER = ':'
```

Comment:
[os.pathsep](https://docs.python.org/2/library/os.html#os.pathsep) can probably be used